### PR TITLE
Edits to Schema and Policies section 

### PR DESCRIPTION
**Pull request by kraigb**
Editing of all these topics; patched up links into those where the section names have been changed. project.lock.json content merged into
project.json (added rerdirect). Also updated Consume workflow graphics.

Note that section on prohibited practices that was in Policies/DisputeResolution.md is now in Deleting-Packages.md, because it belongs with that subject (it wasn't about dispute resolution
specifically).

Also tweaked nuget pack -properties reference.

The mention of $references$ in the nuspec replacement tokens section is removed for the time being, pending discussion with the product team on whether it actually does anything.

### CODE REVIEW

File: NuGet.Docs/ndocs/Policies/Dispute-Resolution.md [hunk 1]
```diff
@@ -1,49 +1,16 @@
...
-
-In both scenarios Alan and Nancy do not appear to be acting with bad intentions, rather they are supporting these tools by contributing their own time and code to make improvements and help other customers of Northwind and AdventureWorks. 
-
-To date, every dispute brought to NuGet's attention has been resolved between the individual parties without NuGet administrators needing to get involved.  Many of these disputes have been resolved between organizations and developers because they share the same goal: make a developers job easier.
-
-The NuGet ecosystem is moving at a fast moving pace, and everyone wants their projects to “just work”.  When people collaborate to find the best solution, it usually works out much better than if a third party gets involved and makes a judgement.  We know there are very good lawyers at many organizations but it is our belief that when developers communicate with other developers a better technical solution for the community can be achieved.
-
-## Changes
+## Process
```
Comment (karann-msft) [inline NuGet.Docs/ndocs/Policies/Dispute-Resolution.md]: Where is this documented in the updated version?

File: .vs/config/applicationhost.config [hunk 2]
```diff
@@ -137,8 +137,8 @@
             <add name="Clr2IntegratedAppPool" managedRuntimeVersion="v2.0" managedPipelineMode="Integrated" CLRConfigFile="%IIS_USER_HOME%\config\aspnet.config" autoStart="true" />
             <add name="Clr2ClassicAppPool" managedRuntimeVersion="v2.0" managedPipelineMode="Classic" CLRConfigFile="%IIS_USER_HOME%\config\aspnet.config" autoStart="true" />
             <add name="UnmanagedClassicAppPool" managedRuntimeVersion="" managedPipelineMode="Classic" autoStart="true" />
-            <applicationPoolDefaults managedRuntimeLoader="v4.0" >
-                <processModel/>
+            <applicationPoolDefaults managedRuntimeLoader="v4.0">
```
Comment (harikmenon) [inline .vs/config/applicationhost.config]: Don't commit this file. Remove this file from the merge.

File: NuGet.Docs/ndocs/Policies/Deleting-Packages.md [hunk 3]
```diff
@@ -1,15 +1,24 @@
 # Deleting packages
 
-NuGet.org does not support permanent deletion of packages, because that would break anyone who is depending on it
-remaining available. This is particularly true when using the workflow that restores packages on build.
-
-Instead, NuGet.org supports a way to 'unlist' a package, which can be done in the package management page on the
-web site. When a package is unlisted, it no longer shows up in search and in any package listing, both on NuGet.org
-and from the NuGet Visual Studio extension (or nuget.exe). However, it remains downloadable by specifying its exact
-version, which is what allows the Restore workflow to continue working.
+NuGet.org does not support permanent deletion of packages. Doing so would break every project depending on the availabiliyt of the package, especially with build workflows that involve package restore.
```
Comment (harikmenon) [inline NuGet.Docs/ndocs/Policies/Deleting-Packages.md]: typo

File: NuGet.Docs/ndocs/Policies/Governance.md [hunk 4]
```diff
@@ -1,60 +1,71 @@
...
 
 In many ways, the role of the benevolent dictator is less about dictatorship and more about diplomacy. The key is to ensure that, as the project expands, the right people are given influence over it and the community rallies behind the vision of the project lead. The lead’s job is then to ensure that the committers (see below) make the right decisions on behalf of the project. Generally speaking, as long as the committers are aligned with the project’s strategy, the project lead will allow them to proceed as they desire.
 
-Additionally, .NET Foundation staff consider the project lead primary point of contact or first point of contact for the project for purposes of business operations including domain registrations, and technical services (e.g. code-signing).
+Additionally, .NET Foundation staff consider the project lead the primary or  first point of contact for NuGet for purposes of business operations including domain registrations, and technical services (e.g. code-signing).
```
Comment (harikmenon) [inline NuGet.Docs/ndocs/Policies/Governance.md]: ,

File: NuGet.Docs/ndocs/Schema/nuget.config-file.md [hunk 5]
```diff
@@ -1,239 +1,290 @@
...
+<a name="repositoryPath"></a>
+<a name="proxy-settings"></a>
+
+Contains miscellaneous configuration settings, which can be set using the [`nuget config` command](/ndocs/tools/nuget.exe-cli-reference#config).
+
+Note: `dependencyVersion` and `repositoryPath` apply only to projects using `packages.config`. `globalPackagesFolder` applies only to projects using `project.json`.
+
+<table>
+	<tr>
+		<th>Key</th>
```
Comment (harikmenon) [inline NuGet.Docs/ndocs/Schema/nuget.config-file.md]: missing defaultpushsource

File: .vs/config/applicationhost.config (hunk 2 above)
Comment (kraigb) [inline .vs/config/applicationhost.config]: Can you tell me how to do that. I see an answer like this, http://stackoverflow.com/questions/12481639/remove-files-from-git-commit, but I've never used git reset and would be flying blind with it.

File: NuGet.Docs/ndocs/Policies/Dispute-Resolution.md (hunk 1 above)
Comment (kraigb) [inline NuGet.Docs/ndocs/Policies/Dispute-Resolution.md]: It's essentially in the fourth paragraph of the edits, just greatly simplified. Are there parts that you think are still essential? The collaboration concept is there, as is the idea that the NuGet team probably doesn't need to be involved. The part about the ecosystem moving at a fast pace would have to be qualified with proof, and it's not necessary to the point of the policy. The part about lawyers seems unnecessary.

File: NuGet.Docs/ndocs/Schema/nuget.config-file.md (hunk 5 above)
Comment (kraigb) [inline NuGet.Docs/ndocs/Schema/nuget.config-file.md]: Added, please check for accuracy.

File: NuGet.Docs/ndocs/Policies/Dispute-Resolution.md (hunk 1 above)
Comment (karann-msft) [inline NuGet.Docs/ndocs/Policies/Dispute-Resolution.md]: Fair enough.

**Pull request merged**